Searching Algo/Dfs.java

```diff
@@ -0,0 +1,58 @@
+/**
+Time Complexity Analysis : I the worst case you would need to visit
+all the vertex and edge hence the time complexity in the worst case is O(V+E).
+
+While Space complexity for DFS is O(h) where h is the maximum height of the tree.
+**/
+
+import java.util.*;
+public class DFS
+{
+int V;    
+LinkedList<Integer> adjList[]; 
+DFS(int v)
+{
```

Comment:
@Knlsharma Where is the indentation. All program is in the single line indentation.